docs/querying/sql-functions.md

```diff
@@ -2473,28 +2575,92 @@ Shifts a timestamp forwards or backwards by a given number of time units.
 
 ## TIMESTAMP_TO_MILLIS
 
-`TIMESTAMP_TO_MILLIS(<TIMESTAMP>)`
+Returns the number of milliseconds since epoch for the given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMP_TO_MILLIS(timestamp_expr)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example converts the `__time` timestamp column from the `taxi-trips` datasource into milliseconds since epoch.
+
+```sql
+SELECT 
+  "__time" AS "original_time",
+  TIMESTAMP_TO_MILLIS("__time") AS "miliseconds"
+FROM "taxi-trips"
+LIMIT 1
+```
+
+Returns the following:
+
+| `original_time` | `miliseconds` |
+| -- | -- |
+| `2013-08-01T08:14:37.000Z` | `1375344877000` |
+
+</details>
+
+
+[Learn more](sql-scalar.md#date-and-time-functions)
 
-Returns the number of milliseconds since epoch for the given timestamp.
 
 ## TIMESTAMPADD
 
-`TIMESTAMPADD(<unit>, <count>, <TIMESTAMP>)`
+Adds `count` number of a `unit` to a given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMPADD(unit, count, timestamp)`
+* **Function type:** Scalar, date and time
 
-Adds a certain amount of time to a given timestamp.
+<details><summary>Example</summary>
 
-## TIMESTAMPDIFF
+The following example adds five months to the timestamp `2000-01-01 00:00:00`
 
-`TIMESTAMPDIFF(<unit>, <TIMESTAMP>, <TIMESTAMP>)`
+```sql
+SELECT
+  TIMESTAMP '2000-01-01 00:00:00' AS "original_time",
+  TIMESTAMPADD (MONTH, 5, TIMESTAMP '2000-01-01 00:00:00') AS "new_time"
+```
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+Returns the following:
+
+| `original_time` | `new_time` |
+| -- | -- |
+| `2000-01-01T00:00:00.000Z` | `2000-06-01T00:00:00.000Z` |
+
+</details>
+
+[Learn more](sql-scalar.md#date-and-time-functions)
+
+## TIMESTAMPDIFF
 
 Takes the difference between two timestamps, returning the results in the given units.
 
+* **Syntax:** `TIMESTAMPDIFF(unit, timestamp1, timestamp2)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example takes the difference between the `__time` and `dropoff_datetime` columns from the `taxi-trips` datasource to find the length of the taxi trip in minutes.
```

Comment:
```suggestion
The following example calculates the taxi trip length in minutes by subtracting the `__time` column from the `dropoff_datetime` column in the `taxi-trips` datasource.
```
Rearranged the text to improve readability. I'm assuming that the function subtracts one value from another?

Reply:
That is how the function works

Reply:
I agree, your example improves readablility